api/pkg/backend/service.go

```diff
@@ -461,9 +461,117 @@ func (s *APIService) listStorageType(ctx context.Context, request *restful.Reque
 	return storageTypes, err
 }
 
+func (s *APIService) IsDuplicateItemExist(itemList []string) []string {
```

Comment:
Should we include all these functions in utils?

Reply:
PR https://github.com/sodafoundation/multi-cloud/pull/1276 has put it in utils

Reply:
done. Thanks